tracim/tracim/config/app_cfg.py

```diff
@@ -217,6 +217,10 @@ def __init__(self):
                 'email.notification.from.default_label.'
             )
 
+        self.AUTO_GENERATED_PASSWORD_LENGTH = tg.config.get(
+            'tracim.password.length'
```

Comment:
Two strategies:

1. you make all parameterizable, so default password length is defined in the configuration file. In this case, allowed chars must be also in the config file.

In this first case, lets call parameters : 
```
tracim.password.length.default` = 8
 tracim.password.chars.default = abcdef... etc
```

Also add these parameters to the template config file

2. You consider this part of the code does not require to be parameterizable. In this case, lets create 2 constants which are : 
DEFAULT_PASSWORD_LENGTH = 8
DEFAULT_PASSWORD_CHARS = 'abcd...

I let you decide wether to parameterize the default password or not (I would do not on my own, but both solutions are viable) and make the fixes


Also